app/src/context/DocumentStore.tsx

```diff
@@ -112,19 +117,52 @@ export const DocumentProvider: React.FC<{
     setFiles(updatedFiles);
   };
 
-  const reset = () => {
+  const resetFiles = () => {
     fileLocalStorage.clear();
     setFiles([]);
   };
 
+  const addDocument = async (_document: Document): Promise<string> => {
+    const id = uuid.v4() as string;
```

Comment:
as string? does this not return string?

Reply:
v4() method return type is `string | number[]`